docs/en/rules/Azure.Automation.PlatformLogs.md

```diff
@@ -0,0 +1,224 @@
+---
+severity: Important
+pillar: Security
```

Comment:
Maybe this should be a different pillar?

Reply:
nice spot, will change this. Thanks 🙂